Formula/zookeeper.rb

```diff
@@ -74,7 +74,8 @@ def install
     inreplace "conf/zoo_sample.cfg",
               /^dataDir=.*/, "dataDir=#{var}/run/zookeeper/data"
     cp "conf/zoo_sample.cfg", "conf/zoo.cfg"
-    (etc/"zookeeper").install ["conf/zoo.cfg", "conf/zoo_sample.cfg"]
+    cp "conf/log4j.properties", "conf/log4j_sample.properties"
```

Comment:
This makes a little more sense to me.
```suggestion
    cp "conf/log4j.properties", "conf/log4j.properties.sample"
```

Reply:
Fixed as you suggested. Thank you.